template/src/components/TextInput/TextInput.tsx

```diff
@@ -36,6 +37,8 @@ type ComponentProps = TextInputProps &
   LayoutProps &
   FlexProps;
 
+const InputContainer = styled(Container)``;
```

Comment:
If there is a better way to do this and allow overriding the values, I would love to pair up to walk through that!